### DISCUSSION TITLE
Upload chunks/0/3/3127_0_62876: bad_request: Checksum did not match data received (try 1) [cached_store.go:396]

### DISCUSSION THREAD

**Discussion by joostfaassen** [juicedata/juicefs#2565] [General]
I'm frequently seeing the following message in juicefs's log:

```
Upload chunks/0/3/3127_0_62876: bad_request: Checksum did not match data received (try 1) [cached_store.go:396]
```

Configuration:

* local redis for metadata
* remote backblaze (on a relatively slow connection) as object storage
* rsa client side encryption.
* happens regardless of  --writeback being used

What does the message mean exactly? Is it a warning signal for data corruption, or just a cache-miss? Is there anything I can do to further debug/resolve the issue?

**Comment by davies** [comment]
This message come from the library talking to B2, I think it means data corruption from network, and the server detect it and JuiceFS will upload that again, so don't worry.
